specification/purview/data-plane/Azure.Analytics.Purview.Account/preview/2023-10-01-preview/account.json

```diff
@@ -0,0 +1,2681 @@
+        "resourceSetProcessing": {
+          "description": "The advanced resource property of the account.",
+          "enum": [
+            "Default",
+            "Advanced"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "resourceSetProcessing",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "BatchFeatureRequest": {
+      "description": "Feature request model",
+      "type": "object",
+      "properties": {
+        "features": {
+          "description": "Set of features",
+          "uniqueItems": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "BatchFeatureStatus": {
+      "description": "List of features with enabled status",
+      "type": "object",
+      "properties": {
+        "features": {
+          "description": "Features with enabled status",
+          "type": "object",
+          "additionalProperties": {
+            "type": "boolean"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "CloudConnectors": {
+      "description": "External Cloud Service connectors",
+      "type": "object",
+      "properties": {
+        "awsExternalId": {
+          "description": "AWS external identifier.\r\nConfigured in AWS to allow use of the role arn used for scanning",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Collection": {
+      "description": "Collection resource.",
+      "type": "object",
+      "properties": {
+        "collectionProvisioningState": {
+          "description": "Gets the state of the provisioning.",
+          "enum": [
+            "Unknown",
+            "Creating",
+            "Moving",
+            "Deleting",
+            "Failed",
+            "Succeeded"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "CollectionProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "description": {
+          "description": "Gets or sets the description.",
+          "type": "string"
+        },
+        "domain": {
+          "description": "Gets the domain.",
+          "type": "string",
+          "readOnly": true
+        },
+        "friendlyName": {
+          "description": "Gets or sets the friendly name of the collection.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Gets the name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "parentCollection": {
+          "$ref": "#/definitions/CollectionReference"
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData"
+        }
+      }
+    },
+    "CollectionList": {
+      "description": "Paged list of collections.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "count": {
```

Comment:
Please consider removing `count` because your service won't be able to page large collections (easily) if it needs to always calculate the total count instead of streaming the list page by page.